Slightly refined documentation while reading it (#1106)
python-attrs/attrs: docs/init.md

ValueError: 'x' has to be smaller than 'y'!
```

> This example shows a syntactic speciality for the {obj}`attrs.validators.and_` validator: If you pass a list, all validators in it have to pass.

Reviewer: I've seen y'all's discussion, but I think I prefer a third option:

```suggestion
This example demonstrates a convenience shortcut:
Passing a list of validators directly is equivalent to passing them wrapped in the {obj}`attrs.validators.and_` validator and all validators must pass.
```

> In this example, the list of validators passed to the `validator` argument, is syntactic sugar for the {obj}`attrs.validators.and_` validator: All validators in the list have to pass.

Reviewer 2: I think I liked the former formulation better, since it isn't the list that is _syntactic sugar_, but the usage of the validator.

Author: Indeed that was the most difficult. 

Because I understood the example (that all validators in the list should pass), I could not understand "why I read this sentence?". 

It took me some time to grasp the connection between the list of validators and `attrs.validatos.and_`.

What about:

>This example is also showing an alternative use (i.e. syntactic sugar) for the {obj}`attrs.validators.and_` validator: 
All validators in the list have to pass.

Reviewer 2: > What about:
> 
> > This example is also showing an alternative use (i.e. syntactic sugar) for the {obj}`attrs.validators.and_` validator:
> > All validators in the list have to pass.

I'd suggest: 

> This example shows a syntactic speciality for the {obj}`attrs.validators.and_` validator: if you pass a list, all validators in it have to pass.

Author: That is very good, I create a commit with it, thank you!

## Converters

> Sometimes, is necessary to normalize the values coming in, therefore *attrs* comes with converters.

Reviewer: ```suggestion
Sometimes, it is necessary to normalize the values coming in, therefore *attrs* comes with converters.
```

## Hooking Yourself Into Initialization

> Generally speaking, the moment you realize the need of finer control -than what *attrs* offers- over how a class is instantiated, it's usually best to use a {obj}`classmethod` factory or to apply the [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern).

Reviewer: We use space-endash-space:

```suggestion
Generally speaking, the moment you realize the need of finer control – than what *attrs* offers – over how a class is instantiated, it's usually best to use a {obj}`classmethod` factory or to apply the [builder pattern](https://en.wikipedia.org/wiki/Builder_pattern).
```
